raiden_contracts/deploy/contract_verifyer.py

```diff
@@ -143,8 +145,10 @@ def _verify_deployment_data(
         chain_id = int(self.web3.version.network)
         assert deployment_data is not None
 
-        assert self.contract_manager.version_string() == deployment_data['contracts_version']
-        assert chain_id == deployment_data['chain_id']
+        if self.contract_manager.version_string() != deployment_data['contracts_version']:
```

Comment:
`version_string`, if you need it dynamically generated, should be a property instead of a method.

Reply:
That's a good idea. I'll fix this in this PR.